routes/searchBarRouter.js

```diff
@@ -2,10 +2,10 @@ import { Router } from "express";
 
 import { tokenValidation } from "../middlewares/tokenValidation.js";
 
-import { getUserBySearch } from "../controllers/searchBarController.js";
+import { getUserBySearch } from "../controllers/userController.js";
 
 const searchBarRouter = Router();
 
-searchBarRouter.get("/searchBar/:name", tokenValidation, getUserBySearch);
+searchBarRouter.get("/search/:nameSearched", tokenValidation, getUserBySearch);
```

Comment:
delete the searchRouter and add the code in the userRouter. This whole task is related to the userRouter.